Add/organize introductions to guidelines chapters (#3855)
qgis/QGIS-Documentation: source/docs/documentation_guidelines/first_contribution.rst

If you are reading these lines, it is certainly because you are willing
to contribute to writing QGIS documentation and are looking for a how-to.

> You are at the right place! The current document will guide you through

Reviewer: ```suggestion
You have come to the right place! The current document will guide you through
```

Let's now dive into the process.

> Documentation sources are stored using a git version control system and

Reviewer: ```suggestion
Documentation sources are stored using the git version control system and are
```

available on GitHub at https://github.com/qgis/QGIS-Documentation.

> You may find two main ways, not mutually exclusive, to modify the files:

Reviewer: ```suggestion
There are two main ways, not mutually exclusive, to modify the files:
```
